build/processes/utils.js

```diff
@@ -8,15 +8,39 @@ class Utils {
   /**
    * Reads a JSON file, removes comments and parses it
    * @param {string} file - path to json file
+   * @param {object} options - options to process resulting JSON
    * @return {object}
    */
-  static readJSON(file) {
-    return JSON.parse(
-      stripJsonComments(
-        fs.readFileSync(file, 'utf8')
-      )
-    );
+  static readJSON(file, options = {}) {
+    if (options.showProgress) {
+      process.stdout.write('.');
+    }
+    let result = stripJsonComments(fs.readFileSync(file, 'utf8'));
+    if (options.lowerCase) {
+      result = result.toLowerCase();
+    }
+    return JSON.parse(result); 
   }
+
+  /**
+   * Writes out a JSON file
+   * @param {string} path - path to JSON file
+   * @param {string} name - JSON file name
+   * @param {object|string} data - JSON file data
+   */
+  static writeJSON(path, name, data) {
```

Comment:
Since you've added this method I just want to hint that there's still a `fs.writeFileSync(` in the build